generators/app/prompt.js

```diff
@@ -201,12 +201,13 @@ function applicationName(obj) {
 }
 
 function target(obj) {
+   var isKube = obj.hasOwnProperty('kube');
    return {
       name: `target`,
       type: `list`,
       store: true,
       message: `Where would you like to deploy?`,
-      choices: util.getTargets,
+      choices: isKube ? util.getKubeTargets :util.getTargets,
```

Comment:
i don't think we should make decisions on which prompt to show here, i think it should be within getTargets